source/AI.cpp

```diff
@@ -1539,6 +1543,20 @@ shared_ptr<Ship> AI::FindTarget(const Ship &ship) const
 				&& foe->IsDisabled() && (!canPlunder || Has(ship, foe->shared_from_this(), ShipEvent::BOARD)))
 			continue;
 
+		foe->UpdateTargeterStrength();
+		double targeterStrength = foe->GetTargeterStrength();
+		int targeterCount = foe->GetShipsTargetingThis().size();
+
+		// The next two checks only apply if more than two ships are attacking the foe, as well as
+		// if it has not already been selected as a target recently.
+		// Deprioritize this if more than a quarter of your allies' strength is already attacking.
+		if(targeterStrength >= 0.25 * alliedStrength && targeterCount > 2 && !preferredTarget)
```

Comment:
I think three to one odds is more sensible.

```suggestion
		if(targeterStrength >= 0.25 * alliedStrength && targeterCount > 3 && !preferredTarget)
```